src/lib/statistics/statistics_objects/min_max_filter.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <iostream>
+#include <memory>
+#include <optional>
+
+#include "abstract_statistics_object.hpp"
+#include "all_type_variant.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ *  Filter that stores a segment's minimum and maximum value
+*/
+template <typename T>
+class MinMaxFilter : public AbstractStatisticsObject {
```

Comment:
This could use some more documentation. What is it good for, why don't we keep the number of entries (because then it would be a one-bucket histogram?)?

Reply:
legacy. I wouldn't want to work on the pruning filters - which were mostly just copied into a different folder - in this PR